scripts/check-dependencies.js

```diff
@@ -1,4 +1,5 @@
-import { checkDependencies } from './utils/cli-utils';
+// eslint-disable-next-line import/extensions
+import { checkDependencies } from './utils/cli-utils.js';
 
 checkDependencies().catch((e) => {
   // eslint-disable-next-line no-console
```

Comment:
We should inline the`checkDependencies` into this file here.
Also add a code-comment stating why this must remain pure JS over TS.

Otherwise, we'll have a mix of js & ts in the utils directory, and it will cause uncertainty as to why.